src/geneweaver/client/api/aon.py

```diff
@@ -0,0 +1,66 @@
+"""Functions that wrap the GeneWeaver API on /genesets endpoints."""
+
+from enum import Enum
+from typing import List, Optional
+
+from geneweaver.client.api.utils import sessionmanager
+from geneweaver.client.core.config import settings
+from geneweaver.core.enum import Species
+
+
+class OrthologAlgorithms(Enum):
+    """The available ortholog algorithms in Geneweaver AON."""
+
+    HGNC = "HGNC"
+    PANTHER = "PANTHER"
+    HIERANOID = "Hieranoid"
+    PHYLOMEDB = "PhylomeDB"
+    ORTHOINSPECTOR = "OrthoInspector"
+    INPARANOID = "InParanoid"
+    ORTHOFINDER = "OrthoFinder"
+    ZFIN = "ZFIN"
+    ENSEMBL_COMPARA = "Ensembl Compara"
+    SONICPARANOID = "SonicParanoid"
+    OMA = "OMA"
+    XENBASE = "Xenbase"
+
+
+def ortholog_mapping(
+    identifiers: List[str], to_species: Species, algorithm_id: Optional[int] = None
```

Comment:
Why not have a default algorithm?

Reply:
Two reasons.

1) After discussing with the stakeholder, it's not clear what the default should be.
2) No algorithm (`algorithm_id=None`) represents a valid call of the API. It represents an API request which doesn't filter the response by algorithm ID, which functionally returns mappings from **all algorithms**.